src/components/drawer/ComponentDetailPanel.vue

```diff
@@ -25,106 +25,82 @@
-          <template
-            v-for="localAttribute in localAttributes"
-            :key="`${localAttribute.title}-${Math.random()}`"
-          >
-
-            <q-separator/>
-
-            <q-expansion-item
-              :default-opened="localAttribute.expanded"
-              :label="$t(localAttribute.title)"
-              class="text-bold"
-            >
 
-              <q-separator/>
-
-              <q-item-section>
-                <div
-                  v-for="attribute in
-                    getSelectedComponentAttributes(localAttribute.attributeKey)"
-                  :key="`${attribute.type}-${Math.random()}`"
-                  class="row items-center q-mb-sm"
-                >
-                  <input-wrapper
-                    :attribute="attribute"
-                    :plugin="localPlugin"
-                    class="col"
-                    @update:attribute-name="(name) => attribute.name = name"
-                    @update:attribute-value="(value) => attribute.value = value"
-                  />
-                  <q-btn
-                    v-if="localAttribute.attributeKey == 'unreferenced'"
-                    class="q-mr-md"
-                    size="xs"
-                    round
-                    flat
-                    color="negative"
-                    icon="fa-solid fa-trash"
-                    data-cy="object-details-panel-attribute-delete-button"
-                    @click="deleteAttribute(attribute.name)"
-                  >
-                    <q-tooltip anchor="center left" self="center right">
-                      {{$t('plugin.component.attribute.delete')}}
-                    </q-tooltip>
-                  </q-btn>
-                </div>
-                <div
-                  v-if="getSelectedComponentAttributes(localAttribute.attributeKey).length === 0"
-                  class="text-grey text-weight-regular q-py-sm q-px-md"
-                >
-                  {{ $t('plugin.component.attribute.noAttributes') }}
-                </div>
-                <q-btn
-                  v-if="localAttribute.attributeKey == 'unreferenced'"
-                  no-caps
-                  class="q-my-md self-center"
-                  :label="$t('plugin.component.attribute.add')"
-                  color="positive"
-                  icon="fa-solid fa-plus"
-                  data-cy="object-details-panel-attribute-add-button"
-                  @click="addAttribute"
-                />
-              </q-item-section>
-            </q-expansion-item>
-
-            <q-separator/>
-
-          </template>
-        <div class="row justify-evenly q-mt-md">
-          <q-btn
-            icon="fa-solid fa-floppy-disk"
-            :label="$t('plugin.component.attribute.save')"
-            type="submit"
-            color="positive"
-            :loading="submitting"
-            data-cy="object-details-panel-save-button"
+    <div class="col" v-if="selectedComponent && isVisible">
```

Comment:
put it in each separate line and `v-if` first

Reply:
done